src/Aspire.Hosting/Nats/NatsBuilderExtensions.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Aspire.Components.Common;
```

Comment:
@radical - can we change the namespace of the `ContainerImageTags` class? I should have caught this in the other PR.

I don't like it being called `Aspire.Components` because these aren't what we are calling "Components". I think it should just be in `namespace Aspire.Hosting;`